recipes/skaware/skalibs_2.3.9.0.oe

```diff
@@ -0,0 +1,5 @@
+require ${PN}.inc
+
+# Workaround for bugs in skalibs, assuming 'char' is 'signed char'.
+# This should/could be dropped when skalibs has fixed these issues.
```

Comment:
Yes, I did report this upstream.  But seems like the mail archvies are incomplete, so I cannot find it there.

This is what I got back:

>On 14/01/2016 18:37, Esben Haabendal wrote:
>> Hi
>>
>> I have fix for a small bug in sgetopt_r().  In C, 'char' type can be
>> either signed or unsigned, and relying on it being one of them is a
>> bug.
>
>  Hi Esben,
>  Thanks a lot! I'm precisely working on ARM these days, and looking
>for this kind of bugs. I'll use an int instead (signed char is an
>icky type).
>  In the meantime, please use CFLAGS=-fsigned-char when compiling
>on ARM. It's the easiest workaround until all those bugs are squashed.
>
>  Cheers, Laurent

And should be fixed in 2.3.10.0, so this can be dropped when upgrade is done.